src/Tests/Moryx.Runtime.Kernel.Tests/Dummys/RuntimeConfigManagerTestConfig1.cs

```diff
@@ -144,6 +144,11 @@ public void RaiseConfigChanged(params string[] modifiedProperties)
                 ConfigChanged(this, new ConfigChangedEventArgs(modifiedProperties));
         }
 
+        public void Initialize()
+        {
+            throw new NotImplementedException();
```

Comment:
Does it work if an exception was thrown? 😅

Reply:
In a test mock for this test: yes :-)